pdr_backend/ppss/web3_pp.py

```diff
@@ -336,6 +336,9 @@ def get_current_epoch(self) -> int:
     def set_token(self, web3_pp):
         pass
 
+    def s_start_payouts(self) -> int:
```

Comment:
This looks like it's an addition to a mock predictoor_contract object. Not the correct place. If it were to be added, it should be an addition to a mock predictoor_ss object